### PR TITLE
DOC-731: Document additional TableModified event data

### PR DESCRIPTION
**Pull request by JamesToohey**
**Related Ticket:** 

DOC-731

**Description of Changes:**

Documents the `structure` and `style` variables in the `TableModified` event.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- ~~[ ] `_data/nav.yml` has been updated (if applicable)~~
- ~~[ ] Files has been included where required (if applicable)~~
- ~~[ ] Files removed have been deleted, not just excluded from the build (if applicable)~~
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by lnewson — APPROVED** [review]

File: _includes/events/table-events.md [hunk 1]
```diff
@@ -6,5 +6,5 @@ The following events are provided by the [Table plugin]({{ site.baseurl }}/plugi
 | newcell | N/A | Fired when a cell is created |
 | ObjectResizeStart | N/A | Fired when a resize action is started on a table, row, column or cell using the resize bars |
 | ObjectResized | N/A | Fired when a resize action is finished on a table, row, column or cell using the resize bars |
-| TableModified | `{ table: HTMLTableElement }` | Fired when style or structural changes are made to a table. <br/>{{site.requires_5_6v}} |
-| TableSelectionClear | N/A  | Fired when the table selection is cleared. <br/>{{ site.requires_5_1v }} |
\ No newline at end of file
+| TableModified | `{ table: HTMLTableElement, structure: boolean, style: boolean }` | Fired when style or structural changes are made to a table. <br/>{{site.requires_5_6v}} |
```
Comment (lnewson) [inline _includes/events/table-events.md:9]: Do we need to say this requires 5.7 now? If not maybe `structure` and `style` should be marked as optional?

Comment (JamesToohey) [reply _includes/events/table-events.md:9]: My personal preference would be to go with 5.7

Comment (lnewson) [reply _includes/events/table-events.md:9]: Yeah, I was leaning that way as well. @tylerkelly13 thoughts?

**Review by EkimChau — APPROVED** [review]

File: _includes/events/table-events.md (hunk 1 above)
Comment (tylerkelly13) [reply _includes/events/table-events.md:9]: this is why we need better versioning of the docs....
let's bump it to 5.7.

**Review by ltrouton — APPROVED** [review]

**Review by tylerkelly13 — APPROVED** [review]

**Pull request merged**